flytepropeller/pkg/controller/config/config.go

```diff
@@ -120,47 +124,94 @@ var (
 			EventVersion:               0,
 			DefaultParallelismBehavior: ParallelismBehaviorUnlimited,
 		},
+		LiteralOffloadingConfig: LiteralOffloadingConfig{
+			Enabled: true, // Default keep this enabled and in case of issues allow users to disable using config map.
```

Comment:
until we have the SDK version checked in, I think we should keep this false just in case it ends up taking multiple releases for the expected SDK change